error in output-reports config

using detekt 1.10 (` id("io.gitlab.arturbosch.detekt").version("1.10.0")`), and including the config stanza:
```
output-reports:
  active: true
  exclude:
  #  - 'HtmlOutputReport'
  #  - 'TxtOutputReport'
  #  - 'XmlOutputReport'
```

I get the following error:
```
$ ./gradlew detekt

> Task :jobs:detekt FAILED
Property 'output-reports' is misspelled or does not exist.

FAILURE: Build failed with an exception.
```

Also, the documentation shows this configuration is available, but the default config file linked from the documentation does not have it.

Finally, running without this stanza commented yields empty xml reports:
```
$ cat lib/build/reports/detekt/detekt.html
...
<h2>Complexity Report</h2>

<div>
  <ul>
    <li>224 lines of code (loc)</li>
    <li>169 source lines of code (sloc)</li>
    <li>130 logical lines of code (lloc)</li>
    <li>23 comment lines of code (cloc)</li>
    <li>32 cyclomatic complexity (mcc)</li>
    <li>9 cognitive complexity</li>
    <li>0 number of total code smells</li>
    <li>13% comment source ratio</li>
    <li>246 mcc per 1,000 lloc</li>
    <li>0 code smells per 1,000 lloc</li>
  </ul>
</div>

$ cat lib/build/reports/detekt/detekt.xml
<?xml version="1.0" encoding="utf-8"?>
<checkstyle version="4.3">
</checkstyle
```

_Originally posted by @sean-abbott in https://github.com/detekt/detekt/issues/2869#issuecomment-663664025_

--- a/detekt-core/src/main/resources/default-detekt-config.yml
+++ b/detekt-core/src/main/resources/default-detekt-config.yml
@@ -31,6 +31,13 @@ console-reports:
   #  - 'FindingsReport'
      - 'FileBasedFindingsReport'
 
+output-reports:
+  active: true
+  exclude:
+  # - 'TxtOutputReport'
+  # - 'XmlOutputReport'
+  # - 'HtmlOutputReport'
+
 comments:
   active: true
   excludes: ['**/test/**', '**/androidTest/**', '**/commonTest/**', '**/jvmTest/**', '**/jsTest/**', '**/iosTest/**']

--- a/detekt-generator/src/main/kotlin/io/gitlab/arturbosch/detekt/generator/printer/rulesetpage/ConfigPrinter.kt
+++ b/detekt-generator/src/main/kotlin/io/gitlab/arturbosch/detekt/generator/printer/rulesetpage/ConfigPrinter.kt
@@ -23,6 +23,8 @@ object ConfigPrinter : DocumentationPrinter<List<RuleSetPage>> {
             emptyLine()
             yaml { defaultConsoleReportsConfiguration() }
             emptyLine()
+            yaml { defaultOutputReportsConfiguration() }
+            emptyLine()
 
             item.sortedBy { it.ruleSet.name }
                 .forEach { printRuleSet(it.ruleSet, it.rules) }
@@ -106,6 +108,15 @@ object ConfigPrinter : DocumentationPrinter<List<RuleSetPage>> {
            - 'NotificationReport'
         #  - 'FindingsReport'
            - 'FileBasedFindingsReport'
+    """.trimIndent()
+
+    private fun defaultOutputReportsConfiguration(): String = """
+      output-reports:
+        active: true
+        exclude:
+        # - 'TxtOutputReport'
+        # - 'XmlOutputReport'
+        # - 'HtmlOutputReport'
     """.trimIndent()
 
     private fun String.isYamlList() = trim().startsWith("-")

--- a/detekt-generator/src/test/kotlin/io/gitlab/arturbosch/detekt/generator/printer/ConfigPrinterSpec.kt
+++ b/detekt-generator/src/test/kotlin/io/gitlab/arturbosch/detekt/generator/printer/ConfigPrinterSpec.kt
@@ -1,12 +1,11 @@
 package io.gitlab.arturbosch.detekt.generator.printer
 
-import io.github.detekt.test.utils.resource
+import io.github.detekt.test.utils.readResourceContent
 import io.gitlab.arturbosch.detekt.generator.printer.rulesetpage.ConfigPrinter
 import io.gitlab.arturbosch.detekt.generator.util.createRuleSetPage
 import org.assertj.core.api.Assertions.assertThat
 import org.spekframework.spek2.Spek
 import org.spekframework.spek2.style.specification.describe
-import java.io.File
 
 class ConfigPrinterSpec : Spek({
 
@@ -14,9 +13,14 @@ class ConfigPrinterSpec : Spek({
 
         it("prints the correct yaml format") {
             val ruleSetList = listOf(createRuleSetPage())
+            val expectedRulePart = readResourceContent("RuleSetConfig.yml")
+
             val yamlString = ConfigPrinter.print(ruleSetList)
-            val expectedYamlString = File(resource("/RuleSetConfig.yml")).readText()
-            assertThat(yamlString).contains(expectedYamlString)
+
+            assertThat(yamlString).contains(expectedRulePart)
+            assertThat(yamlString).contains("output-reports:")
+            assertThat(yamlString).contains("console-reports:")
+            assertThat(yamlString).contains("processors:")
         }
     }
 })

